spec/support/idv_examples/usps_address_confirmation.rb

```diff
@@ -0,0 +1,52 @@
+shared_examples 'idv usps address confirmation' do |sp|
+  it 'allows the user to select verification via USPS letter', email: true do
+    perform_loa3_request_from_sp(sp)
+
+    user = register_user
+
+    click_idv_begin
+    fill_out_idv_form_ok
+    click_idv_continue
+    fill_out_financial_form_ok
+    click_idv_continue
+
+    click_idv_address_choose_usps
+
+    click_on t('idv.buttons.mail.send')
+
+    expect(current_path).to eq verify_review_path
+    expect(page).to_not have_content t('idv.messages.phone.phone_of_record')
+
+    fill_in :user_password, with: user_password
+
+    expect { click_submit_default }.
+      to change { UspsConfirmation.count }.from(0).to(1)
+
+    expect(current_path).to eq verify_confirmations_path
+    click_acknowledge_personal_key
+
+    user.reload
```

Comment:
This test is almost completely copy/pasted from the original one except for this line through 37. I added those to check that we match the initial state setup in #1581